jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -51,19 +51,25 @@ class MomentMap(PluginTemplateMixin, DatasetSelectMixin, SpectralSubsetSelectMix
     overwrite_warn = Bool(False).tag(sync=True)
 
     def __init__(self, *args, **kwargs):
-        self._default_spectrum_viewer_reference_name = kwargs.get(
-            "spectrum_viewer_reference_name", "spectrum-viewer"
-        )
-        self._default_image_viewer_reference_name = kwargs.get(
-            "image_viewer_reference_name", "image-viewer"
-        )
         super().__init__(*args, **kwargs)
 
         self.moment = None
 
         self.dataset.add_filter('is_cube')
         self.add_results.viewer.filters = ['is_image_viewer']
 
+    @property
+    def _default_image_viewer_reference_name(self):
+        return getattr(
+            self.jdaviz_helper, '_default_image_viewer_reference_name', 'image-viewer'
+        )
+
+    @property
+    def _default_spectrum_viewer_reference_name(self):
+        return getattr(
+            self.jdaviz_helper, '_default_spectrum_viewer_reference_name', 'spectrum-viewer'
+        )
```

Comment:
Can we just call `self.jdaviz_helper._default_..._reference_name` within the plugin or do we need the extra fallback?  If so, maybe we a `get_default_viewer(type)` method at the plugin-mixin level could handle this generically so other plugins could make use of it as well?

Reply:
You're absolutely right – the fallbacks to the old defaults are not needed. I've removed them in [4e3e8b6](https://github.com/spacetelescope/jdaviz/pull/2338/commits/4e3e8b68fc3da5b948e240f97902144faf406623).